assets/css/base/_base.scss

```diff
@@ -85,3 +85,12 @@ button:-moz-focusring {
   color: transparent;
   text-shadow: 0 0 0 $grey-dark;
 }
+
+// News
+
+.news-item {
```

Comment:
Please move the css for news-and-views out of `_base.scss` into a separate "_news-and-views.scss` in the `assets/css/components/` directory.